mesh_map/include/mesh_map/mesh_map.h

```diff
@@ -398,11 +400,19 @@ class MeshMap
    */
   mesh_map::AbstractLayer::Ptr layer(const std::string& layer_name);
 
+  /**
+   * @brief calls 'saveLayer' on every active layer
```

Comment:
maybe add where does it saves the mesh to and mention it's accessible via service